index.md

```diff
@@ -1,19 +1,24 @@
 ---
 layout: default
 ---
-# Hackathon Hackers Europe
-We're hosting a casual meet-up for UK and European hackathon organisers!
+# Hackathon Hackers Europe 
+Hackathon Hackers Europe (or HHEU for short) is an organisation targetting at helping student lead hackathons around europe. We are here to bring the hackatoh organiser and attendee community together to make hackathons easier to run and partipate in.
```

Comment:
Few typos in this sentence, e.g. `targetting`, `hackatoh`, could probably rewrite this with better grammar